### PR TITLE
bpo-35116: Adding Doc/library entries for cgi.FieldStorage max_num_fields

### PR DESCRIPTION
**Pull request by matthewbelisle-wf**
https://bugs.python.org/issue35116

### CODE REVIEW

**Comment by matthewbelisle-wf** [comment]
For your review @vstinner from your comment in https://github.com/python/cpython/pull/9969#issuecomment-434344907.

File: Doc/library/urllib.parse.rst [hunk 1]
```diff
@@ -166,8 +170,11 @@ or on combining URL components into a URL string.
    .. versionchanged:: 3.2
       Add *encoding* and *errors* parameters.
 
+   .. versionchanged:: 3.8
+      Added *max_num_fields* param.
```
Comment (vstinner) [inline Doc/library/urllib.parse.rst]: Please replace "param" with "parameter.

File: Doc/library/urllib.parse.rst [hunk 2]
```diff
@@ -187,12 +194,19 @@ or on combining URL components into a URL string.
...
+   than *max_num_fields* fields read. Default is *None*.
+
    Use the :func:`urllib.parse.urlencode` function to convert such lists of pairs into
    query strings.
 
    .. versionchanged:: 3.2
       Add *encoding* and *errors* parameters.
 
+   .. versionchanged:: 3.8
+      Added *max_num_fields* param.
```
Comment (vstinner) [inline Doc/library/urllib.parse.rst]: Ditto.

**Comment by matthewbelisle-wf** [comment]
Ditto, fixed in 1c3f2f0 @vstinner

File: Doc/library/urllib.parse.rst [hunk 3]
```diff
@@ -158,6 +158,10 @@ or on combining URL components into a URL string.
    percent-encoded sequences into Unicode characters, as accepted by the
    :meth:`bytes.decode` method.
 
+   The optional argument *max_num_fields* is a flag for the maximum number of
```
Comment (vstinner) [inline Doc/library/urllib.parse.rst]: I would just say "The optional argument *max_num_fields* is the maximum number of ..."

Note: wait until this PR is merged before updating the 3 other PRs :-)

File: Doc/library/urllib.parse.rst [hunk 4]
```diff
@@ -158,6 +158,10 @@ or on combining URL components into a URL string.
...
+   fields to read. If set, then throws a :exc:`ValueError` if there are more
+   than *max_num_fields* fields read. Default is *None*.
```
Comment (vstinner) [inline Doc/library/urllib.parse.rst]: "Default is *None*." IMHO it's redundant with the function prototype and can be removed.

File: Doc/library/urllib.parse.rst [hunk 5]
```diff
@@ -187,12 +194,19 @@ or on combining URL components into a URL string.
...
```
Comment (vstinner) [inline Doc/library/urllib.parse.rst]: Ditto.

**Comment by matthewbelisle-wf** [comment]
Sure thing @vstinner fixed in 51d6fb37bf.

**Review by vstinner — APPROVED** [review]
LGTM.

**Comment by vstinner** [comment]
You can now update the 3 other PRs :-)

**Pull request merged**